src/Runner.Worker/StepsRunner.cs

```diff
@@ -195,6 +195,12 @@ public async Task RunAsync(IExecutionContext jobContext)
                         }
                         else
                         {
+                            // This is our last, best chance to expand the display name.  (At this point, all the requirements for successful expansion should be met.)
+                            // That being said, evaluating the display name should still be considered as a "best effort" exercise.  (It's not critical or paramount.)
+                            // For that reason, we call a safe "Try..." wrapper method to ensure that any potential problems we encounter in evaluating the display name
+                            // don't interfere with our ultimate goal within this code block:  evaluation of the condition.
+                            TryUpdateDisplayName(step as IActionRunner, out _);
```

Comment:
Do we need the `out _` var here / in the function signature?

Reply:
yeah, I think so, for consistency.

There are effectively three possible outcomes.

Call succeeded as the result of a successful update to the display name.
Call succeeded because the display name was already up-to-date.
Call failed

We've seen that there are some codepaths (one level down) that need to distinguish between those two flavors of success, so for consistency, I'm preserving that level of detail.

If it's not appealing, the next best alternative would be to drop the out parameter and instead return a bitwise enum that encodes the various flavors of success.  I considered that, but it felt a bit over the top.